test/timing_load_creds.c

```diff
@@ -21,6 +21,19 @@
 # include <openssl/bio.h>
 # include "internal/e_os.h"
 
+# ifndef timersub
+#  define timersub(a, b, res)                                         \
+     do {                                                             \
+         (res)->tv_sec = (a)->tv_sec - (b)->tv_sec;                   \
```

Comment:
Do we assume a <= b? Or do we have to handle possible underflows?

Reply:
The glibc macro does not handle them. IMO it is sufficient as it is.

Reply:
Ok. Perhaps just add a comment before the macro that we don't handle that situation.

Reply:
Done.